tools/spectral/ipa/scripts/generateRulesetReadme.js

```diff
@@ -26,35 +25,58 @@ fs.writeFile(readmeFilePath, fileContent, (error) => {
 
 async function getRulesetsSection() {
   let content =
-    '## Rulesets\n\n' + 'The tables below lists all available rules, their descriptions and severity level.\n\n';
+    '## Rulesets\n\n' + 'Below is a list of all available rules, their descriptions and severity levels.\n\n';
 
   const rules = await getAllRules();
   const ruleNames = Object.keys(rules);
   const ipaNumbers = getIpaNumbers(ruleNames);
 
   ipaNumbers.forEach((ipaNumber) => {
     const ipaRules = filterRulesByIpaNumber(ipaNumber, rules);
-    const table = generateRulesetTable(ipaRules);
-    content +=
-      `### ${ipaNumber}\n\n` + `For rule definitions, see ${getIpaRulesetUrl(ipaNumber)}.\n\n` + `${table}\n\n`;
+    const sections = generateRulesetSections(ipaRules);
+    content += `### ${ipaNumber}\n\n` + `Rule is based on ${getIpaRulesetUrl(ipaNumber)}.\n\n` + `${sections}\n\n`;
   });
 
   return content;
 }
 
-function generateRulesetTable(rules) {
-  const table = [['Rule Name', 'Description', 'Severity']];
-  const tableRows = [];
-
+function generateRulesetSections(rules) {
+  let sections = '';
   const ruleNames = Object.keys(rules);
-  ruleNames.forEach((ruleName) => {
-    const rule = rules[ruleName];
-    tableRows.push([ruleName, rule.description, rule.definition.severity]);
+  const sortedRuleEntries = ruleNames
+    .map((ruleName) => ({
+      name: ruleName,
+      description: rules[ruleName].description,
+      severity: rules[ruleName].definition.severity,
+    }))
+    .sort((a, b) => {
+      if (a.severity < b.severity) {
+        return -1;
+      } else if (a.severity > b.severity) {
+        return 1;
+      }
+      return 0;
+    });
+
+  sortedRuleEntries.forEach((rule) => {
+    const severityFormatted = formatSeverity(rule.severity);
+    sections += `#### ${rule.name}\n\n ${severityFormatted}  ${rule.description}\n\n`;
```

Comment:
```suggestion
    sections += `#### ${rule.name}\n\n ${severityFormatted} \n${rule.description}\n\n`;
```
[Nit] Have the severity icon on a separate line